src/ClientData/include/ClientData/ThreadStateSliceInfo.h

```diff
@@ -5,6 +5,7 @@
 #ifndef CLIENT_DATA_THREAD_STATE_SLICE_INFO_H_
 #define CLIENT_DATA_THREAD_STATE_SLICE_INFO_H_
 
+
```

Comment:
Could you also remove this added line? It's not really necessary for the change.